src/github/handlers.ts

```diff
@@ -69,10 +68,14 @@ export const prHandler = async (context: Context) => {
       return;
     }
 
-    const parsedResult = parseBody(pullRequest.body);
+    const parsedResult = parseResult(pullRequest);
 
-    if (!(parsedResult.board_id && parsedResult.item_id)) {
-      throw new Error('Missing required Monday.com IDs in description!');
+    if (!parsedResult.board_id) {
+      throw new Error('Missing required Monday.com Board ID in PR body!');
+    }
+
+    if (!parsedResult.item_id) {
+      throw new Error('Missing required Monday.com item IDs in Branch name!\n\`<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]\`');
```

Comment:
can you edit this error to:
Missing required Monday.com item IDs in Branch name!\nPlease close this PR and submit a new one with your renamed branch, with naming convention: \`<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]\`